src/ui_parts/code_editor.gd

```diff
@@ -19,6 +19,14 @@ func _ready() -> void:
 	SVG.root_tag.child_attribute_changed.connect(auto_update_text.unbind(1))
 	SVG.root_tag.tag_layout_changed.connect(auto_update_text)
 	SVG.root_tag.changed_unknown.connect(auto_update_text)
+	try_open_with()
+
+func try_open_with() -> void:
+	await get_tree().process_frame
+
+	var cmdline_args := OS.get_cmdline_args()
+	if(cmdline_args.size() > 0):
+		apply_svg_from_path(cmdline_args[0])
```

Comment:
Style nitpick: Remove the parentheses after `if` as per the GDScript style guide. Also, please add a tag whitespace on the empty line (or don't have an empty line, it's a small enough function that it's readable anyway).

This code should probably go in SVG.gd when the SVG text itself is added. And without the await.

Reply:
alright, Ill fix those. C# habbits sneaking in lol

Reply:
Reason its not in svg.gd is because (chances are ive interpreted the code wrong) I think it needs to run on _ready/frame after ready

Reply:
I see what you mean about SVG.gd now, Ill do that

Reply:
Done